imagetest/test_suites/metadata/setup.go

```diff
@@ -22,10 +26,12 @@ done`
 echo "%s" > %s`
 	shutdownOutputPath = "/shutdown_out.txt"
 	shutdownContent    = "The shutdown script worked."
-	shutdownMaxLength  = 32768 // max shutdown metadata value
+	// max metadata value https://cloud.google.com/compute/docs/metadata/setting-custom-metadata#limitations
+	metadataMaxLength = 32768
```

Comment:
Thanks for the link; I'm seeing "Each metadata value has a maximum limit of 256 KB" on that page -- is 32768 something else? Also, in the comment, can you specify the unit? something like: Maximum size of metadata value payload, in bytes.

Reply:
Interesting, i didn't pay much attention on this value. I found this link, it seems these is inconsistent which I think one is legacy value.
https://cloud.google.com/sdk/gcloud/reference/container/clusters/create

Reply:
```Values must be less than or equal to 32,768 bytes in length. The total size of all keys and values must be less than 512 KB.```